lib/auth/session.ts

```diff
@@ -0,0 +1,33 @@
+import MongoStore from "connect-mongo";
+import mongoose from "mongoose";
+import { NextApiRequest, NextApiResponse } from "next";
+import NextSession from "next-session";
+import { promisifyStore } from "next-session/lib/compat";
+
+export default async (
+  req: NextApiRequest,
+  res: NextApiResponse,
+  next: () => void
+) => {
+  const store = promisifyStore(
+    new MongoStore({
+      client: mongoose.connection.getClient(),
+      stringify: false,
+    })
+  );
+
+  const session = NextSession({
+    store,
+    touchAfter: 1000 * 60 * 60 * 24 * 7, // 1 week
+    cookie: {
+      maxAge: 1000 * 60 * 60 * 24 * 7, // 1 week
```

Comment:
Is this the number of milliseconds in a week?

Reply:
Yep!